cmd/node/config/config.toml

```diff
@@ -264,6 +264,10 @@
     [Antiflood.Cache]
         Size = 5000
         Type = "LRU"
+    [Antiflood.Topic]
+        DefaultMaxRequestsPerSec = 75
```

Comment:
Might change the namings as the keyword Requests will be used in antiflooding in a next PR

Reply:
done